lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,22 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+
+    if any(symb.isalpha() for symb in text) is False:
+        return []
+    new_text = []
+    word_for_new_text = []
```

Comment:
Is it really necessary to create two lists?

Reply:
yes?.. How can I do two different actions in ine moment? In my code, I at first put separated letters into the list 'word_for_new_text', when there appears a space (the border between words) I go to the other step. In that step I make these letters to become a word using .join(). After all I place a word into the list 'new_text'.

Now I see the only way to delete 'new_text': to copy the initial one and to clear one of them but it doesn't make any sense

Reply:
done